backvendor/git_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2018 Tim Waugh
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package backvendor
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"golang.org/x/tools/go/vcs"
+)
+
+const (
+	envHelper     = "GO_WANT_HELPER_PROCESS"
+	envStdout     = "STDOUT"
+	envStderr     = "STDERR"
+	envExitStatus = "EXIT_STATUS"
+)
+
+var mockedExitStatus = 0
```

Comment:
No initialisation required if it's zero value, and abnormal to do it.

Reply:
Fixed.